### PR TITLE
1590: Rename environments

### PR DESCRIPTION
**Pull request by shaunharrisonFR**
Rename files and folders for kustomize

Issue: https://github.com/SecureApiGateway/SecureApiGateway/issues/1590

### CODE REVIEW

File: core/kustomize/overlay/openig-nightly/configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: core-deployment-config
+data:
+  BASE_FQDN: "openig-nightly-core.forgerock.financial"
```
Comment (wmorrison365fr) [inline core/kustomize/overlay/openig-nightly/configmap.yaml:6]: nightly? or aic?

Comment (wmorrison365fr) [reply core/kustomize/overlay/openig-nightly/configmap.yaml:6]: Lol, nope... this is the new name that Shaun had questions over.

It's the IG latest - bleeding-edge build.

Comment (shaunharrisonFR) [reply core/kustomize/overlay/openig-nightly/configmap.yaml:6]: I had to double check them myself, In the PR it makes even less sense with the removal of nightly-ob|core, but yeah think its alright

**Review by wmorrison365fr — DISMISSED** [review]

File: test-trusted-directory/kustomize/overlay/dev-aic/kustomization.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-namespace: latest-core
+namespace: dev-aic-core
```
[the remark this answers is not in the document]
Comment (shaunharrisonFR) [reply test-trusted-directory/kustomize/overlay/dev-aic/kustomization.yaml:1]: This looks like Github has gotten confused with what files are changing, notice this is comparing `latest` -> `dev-aic` where it should be `nightly` -> `dev-aic` 

And down below its comparing `nightly` -> `openig-nightly` where it should be `latest` -> `openig-nightly`

**Review by BohoCode — APPROVED** [review]

**Pull request merged**